src/client/app/frameworks/sample/sample.module.ts

```diff
@@ -28,6 +28,10 @@ import { MultilingualModule } from '../i18n/multilingual.module';
   providers: [
     SAMPLE_PROVIDERS
   ],
+  schemas: [
+    NO_ERRORS_SCHEMA,
+    CUSTOM_ELEMENTS_SCHEMA
```

Comment:
http://stackoverflow.com/questions/39428132/custom-elements-schema-added-to-ngmodule-schemas-still-showing-error